src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -32,11 +32,32 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
     })
   }
 
+  handleArtworkShare = async () => {
+    const { title, artist: name, id } = this.props.artwork
+
+    try {
+      const result = await Share.share({
+        message: `${title} by ${name} on Artsy http://artsy.net/artwork/${id}`,
+      })
+      // TODO: Leaving these options here if we're adding analyics
+      if (result.action === Share.sharedAction) {
```

Comment:
I'm not sold on leaving this dead code in here, but we probably will end up tracking it!